src/test/java/org/jenkinsci/plugins/workflow/cps/SnippetizerTest.java

```diff
@@ -301,7 +303,7 @@ public void singleArgStepDocs() throws Exception {
 
     @Test
     public void oneOrMoreArgsStepDocs() throws Exception {
-        SnippetizerTester.assertDocGeneration(InputStep.class);
+        SnippetizerTester.assertDocGeneration(TimeoutStep.class);
```

Comment:
`timeout` has one required parameter (`time`), and one optional parameter (`unit`), which is I think the scenario this test was trying to cover.

Reply:
Perhaps. I do not really recall. Feel free to switch any of these to mock steps using `@TestExtension` if that is clearer or easier.